sass/layout/_flexContainer.scss

```diff
@@ -0,0 +1,7 @@
+.l-flex-container {
```

Comment:
What does the `l` in `.l-flex-container` mean? And why is this file the only one named with camelCase?